core/src/main/scala/org/apache/spark/BarrierCoordinator.scala

```diff
@@ -141,7 +145,7 @@ private[spark] class BarrierCoordinator(
       logInfo(s"Current barrier epoch for $barrierId is $barrierEpoch.")
       if (epoch != barrierEpoch) {
         requester.sendFailure(new SparkException(s"The request to sync of $barrierId with " +
-          s"barrier epoch $barrierEpoch has already finished. Maybe task $taskId is not " +
+          s"barrier epoch $epoch has already finished. Maybe task $taskId is not " +
```

Comment:
During write the UT for ContextBarrierState, I think this is a little bug in log? @jiangxb1987 Please checking if I'm wrong.